Hometask_2

nsubbotina opened the pull request.

nsubbotina committed:
Hometask_2

illarionovEA reviewed (changes requested):
нужно убрать файлы checkpoints из коммита
доделать все задания

nsubbotina committed:
Delete hometask2-checkpoint.ipynb

nsubbotina committed:
k_means

Ссылка на датасет не работала, я постаралась справиться своими силами, нашла этот датасет и сохранила в файл
